ebi_eva_common_pyutils/spring_properties.py

```diff
@@ -26,90 +28,153 @@ def __init__(self, maven_profile, private_settings_file):
+                'spring.batch.job.names': 'EXPORT_SUBMITTED_VARIANTS_JOB',
+                'parameters.taxonomy': taxonomy,
+                'parameters.fasta': fasta,
+                'parameters.assemblyReportUrl': self._format_str('file:{0}', assembly_report),
+                'parameters.projects': projects,
+                'parameters.outputFolder': output_folder
+            })
 
     def get_remapping_ingestion_properties(self, *, source_assembly='', target_assembly='', vcf='', load_to='',
                                            remapping_version=1.0):
         """Properties for remapping ingestion pipeline."""
-        return self._common_properties(target_assembly) + f'''
-spring.batch.job.names=INGEST_REMAPPED_VARIANTS_FROM_VCF_JOB
-parameters.vcf={vcf}
-parameters.remappedFrom={source_assembly}
-parameters.loadTo={load_to}
-parameters.remappingVersion={remapping_version}
-'''
+        return self._format(
+            self._common_properties(target_assembly),
+            {
+                'spring.batch.job.names': 'INGEST_REMAPPED_VARIANTS_FROM_VCF_JOB',
+                'parameters.vcf': vcf,
+                'parameters.remappedFrom': source_assembly,
+                'parameters.loadTo': load_to,
+                'parameters.remappingVersion': remapping_version,
+            }
+        )
 
     def get_clustering_properties(self, *, instance,
                                   job_name='', target_assembly='', rs_report_path='', projects='',
-                                  project_accession='', vcf='', source_assembly=''):
+                                  project_accession='', vcf=''):
         """Properties common to all clustering pipelines, though not all are always used."""
-        return self._common_properties(target_assembly) + self._accessioning_properties(instance) + f'''
-spring.batch.job.names={job_name}
-parameters.projects={projects}
-parameters.projectAccession={project_accession}
-parameters.vcf={vcf}
-parameters.remappedFrom={source_assembly}
-parameters.rsReportPath={rs_report_path}
-'''
+        return self._format(
+            self._common_properties(target_assembly),
+            self._accessioning_properties(instance),
+            {
+                'spring.batch.job.names': job_name,
+                'parameters.projects': projects,
+                'parameters.projectAccession': project_accession,
+                'parameters.vcf': vcf,
+                'parameters.rsReportPath': rs_report_path
+            }
+        )
+
+    def get_release_properties(self, job_name='', assembly_accession='', fasta='', assembly_report='', output_folder=''):
+        if assembly_report is not None:
+            assembly_report = f"{'file:' if assembly_report else ''}{assembly_report}",
+        return self._format(
+            self._common_properties(assembly_accession=assembly_accession),
+            {
+                'spring.batch.job.names': job_name,
+                'parameters.contigNaming': 'INSDC',
```

Comment:
This might need to be a parameter, it's `SEQUENCE_NAME` in the [regular release automation](https://github.com/EBIvariation/eva-accession/blob/master/eva-accession-release-automation/run_release_in_embassy/create_release_properties_file.py#L72)

Reply:
Done